cve_bin_tool/package_list_parser/__init__.py

```diff
@@ -189,9 +211,25 @@ def check_file(self):
                         raise InvalidListError(
                             f"The packages {','.join(not_installed_packages)} seems to be not installed.\nIt is either an invalid package or not installed.\nUse `sudo yum install $(cat package-list)` to install all packages"
                         )
+            elif distro.id() in PACMAN_DISTROS:
+                output = run(
+                    [f"xargs", "-a", input_file, "pacman", "-Qk"],
+                    stderr=PIPE,
+                    stdout=PIPE,
+                )
+
+                not_installed_packages = re.findall(
+                    r"error: package '(.+)' was not found",
+                    output.stderr.decode("utf-8"),
+                )
 
+                if not_installed_packages:
+                    with ErrorHandler(mode=error_mode):
+                        raise InvalidListError(
+                            f"The packages {','.join(not_installed_packages)} seems to be not installed.\nIt is either an invalid package or not installed.\nUse `sudo pacman -S $(cat package-list)` to install all packages"
+                        )
             else:
                 # TODO: Replace below error handling with a proper pip install dry run
                 # See: https://github.com/pypa/pip/issues/53
                 with ErrorHandler(mode=error_mode):
-                    raise InvalidListError("Invalid Python Package list")
+                    raise InvalidListError("Invalid Package list")
```

Comment:
Do we have a way for the user to see on the command line which package managers are supported?  That might be a good future feature.